packages/connect/src/implementation.ts

```diff
@@ -99,14 +102,17 @@ export interface HandlerContext {
 export function createHandlerContext(
   spec: { service: ServiceType; method: MethodInfo },
   deadline: AbortSignal | undefined, // TODO
+  signal: AbortSignal,
   requestHeader: HeadersInit,
   responseHeader: HeadersInit,
   responseTrailer: HeadersInit
 ): HandlerContext {
+  const linkedAbortcontroller = createLinkedAbortController(deadline, signal);
+
   return {
     method: spec.method,
     service: spec.service,
-    deadline,
+    signal: linkedAbortcontroller.signal,
```

Comment:
Good idea to link the signals here 👍 
Can you simply put `createLinkedAbortController(deadline, signal)` on this line?

Reply:
Yes!